build_for_ultimaker.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# SPDX-License-Identifier: AGPL-3.0+
+#
+# Copyright (C) 2019 Ultimaker B.V.
+#
+# SPDX-License-Identifier: LGPL-3.0+
+
+set -eu
+
+ARCH="armhf"
+
+SRC_DIR="$(pwd)"
+RELEASE_VERSION="${RELEASE_VERSION:-999.999.999}"
+DOCKER_WORK_DIR="/build"
+BUILD_DIR_TEMPLATE="_build_${ARCH}"
+BUILD_DIR="${BUILD_DIR:-${SRC_DIR}/${BUILD_DIR_TEMPLATE}}"
+
+run_env_check="yes"
+run_linters="yes"
```

Comment:
No.

Reply:
The run_env_check is just a smoke test for the docker image. The run_linters is just to catch errors in the support scripts. These can be set to no by default or just removed, only the automatic running of XDS tests by default will remain.